### PR TITLE
doc(kafka-rw): refactor data ingestion and analysis from Kafka

### PR DESCRIPTION
**Pull request by WingsUpete**
1. Move the `Analyze the data` part in Section 00-01 to the `Steps` part in Section 00-02.
2. Add a verification paragraph for the `Create a source` part in Section 00-01, by querying a created MV which fetches all data from the source.
3. Add a new `Create a table` part in Section 00-01.

### CODE REVIEW

File: 00-get-started/01-ingest-kafka-data.md [hunk 1]
```diff
@@ -67,38 +67,50 @@ WITH (
...
+WITH (
+ connector='kafka',
+ topic='test',
+ properties.bootstrap.server='localhost:9092',
+ scan.startup.mode='earliest'
+) FORMAT PLAIN ENCODE JSON;
 ```
 
-The results will look like the following. Note that the rows do not necessarily follow this order.
+For verification, run `SELECT * FROM website_visits_table;` and you should see the same outputs in the [`Create a source` part](#create-a-source).
```
Comment (WingsUpete) [inline 00-get-started/01-ingest-kafka-data.md:113]: Maybe also add suggestions on when sources/tables are preferred?

File: 00-get-started/01-ingest-kafka-data.md [hunk 2]
```diff
@@ -67,38 +67,50 @@ WITH (
 ) FORMAT PLAIN ENCODE JSON;
 ```
 
-To learn more about the `CREATE SOURCE` command, check [`CREATE SOURCE`](https://docs.risingwave.com/docs/current/sql-create-source/) from the offical RisingWave documentation.
+Optionally, for verification, you can create a materialized view to grab all existing data from the source, using the following SQL.
```
Comment (st1page) [inline 00-get-started/01-ingest-kafka-data.md:70]:
```suggestion
By creating a source, RisingWave has been connected to the Kafka. However, to ingest the data from Kafka, we need to create some materialized views. Using the following SQL, you can create a materialized view to grab all existing data from the source and continuously capture newly inserted events from the kafka. 
```

File: 00-get-started/01-ingest-kafka-data.md [hunk 3]
```diff
@@ -67,38 +67,50 @@ WITH (
...
+WITH (
+ connector='kafka',
+ topic='test',
+ properties.bootstrap.server='localhost:9092',
+ scan.startup.mode='earliest'
+) FORMAT PLAIN ENCODE JSON;
 ```
 
-The results will look like the following. Note that the rows do not necessarily follow this order.
+For verification, run `SELECT * FROM website_visits_table;` and you should see the same outputs in the [`Create a source` part](#create-a-source).
```
Comment (st1page) [inline 00-get-started/01-ingest-kafka-data.md:112]:
```suggestion
For verification, run `SELECT * FROM website_visits_table;` and you should see the same outputs with selecting from the materialized view created above.
```

File: 00-get-started/01-ingest-kafka-data.md (hunk 1 above)
Comment (st1page) [reply 00-get-started/01-ingest-kafka-data.md:113]: Yes, I will write a pr to compare them later.

**Review by st1page — APPROVED** [review]
Maybe we can add a phase to insert some new record into the kafka and select from the MV/Table again to show that RW can continuously capture newly inserted events. 🤔

**Pull request merged**